04-vector-raster-integration-advanced.Rmd

```diff
@@ -277,6 +293,13 @@ plot(plot.locationSp_HARV,
 
 ```
 
```

Comment:
@mjones01 this is very confusing text. Particularly the last sentence. I am not sure what you are trying to say so i propose we delete it. Stating that the extents are different is sufficient.

+Looking at the plot we created in the challenge above, we see all of the vegetation plot
+locations (blue) layered nicely on top of the Canopy Height Model raster - except for one. 
Why is one location missing? Refer
+back to the first figure in this lesson which shows the extent boundaries of both the 
CHM and the plot locations. Notice that the plot extent (displayed in black) extends slightly farther 
+west than the CHM raster (green) extent.